migrate/starter/starter.go

```diff
@@ -61,6 +62,11 @@ func checkStartVersion(args []string) version {
 	if err != nil {
 		return internalV2
 	}
+	if fs.Lookup("version").Value.String() == "true" {
+		fmt.Println("etcd version:", etcdversion.Version, "(allow legacy mode)")
```

Comment:
It's important that `--version` reference the version number of the binary.  If a user needs to check the version of their data directory that's fine, but the standard flag is not (in my humble opinion) the place to be doing that.